internal/util/kubernetes/unstructured_util.go

```diff
@@ -261,6 +281,12 @@ func CreateResource(ctx context.Context, c client.Client, obj *GenericObject) er
 	return c.Create(ctx, unstructuredObj)
 }
 
+// CreateResourceUnstructured creates the resource in the kubernetes cluster
+// TODO: This is a temporary function which will be removed once all the controller are migrated to use Unstructured Object
```

Comment:
when you say these are "temporary functions", do you mean that the logic for them will still exist, but basically they will be renamed as the original function names? (e.g. `CreateResource()`)

Reply:
Right